static/monorail-2.0.yaml

```diff
@@ -2321,7 +2321,106 @@ paths:
           description: Unexpected error
           schema:
             $ref: '#/definitions/Error'
-
+  /nodes/{identifier}/relations:
+    x-swagger-router-controller: nodes
+    get:
+      operationId: nodesGetRelations
+      x-privileges: [ 'Read' ]
+      x-authentication-type: [ 'jwt' ]
+      summary: |
+        List of all relations for a node or if there are none an empty object
+      description: |
+        List of all relations for a node or if there are none an empty object
+      parameters:
+        - name: identifier
+          in: path
+          description: |
+            Mac addresses and unique aliases to identify the node by, |
```

Comment:
maybe it could be simple and be consistent with other routes like https://github.com/RackHD/on-http/blob/master/static/monorail-2.0.yaml#L2037